### ISSUE TITLE
BLDC Motor control

### ISSUE THREAD

**Issue by bhargavah** [OpenEtherCATsociety/SOEM#578]
Hi Guys,

This is not an issue, instead of a question. I haven't seen any questions on this topic here.

My idea is to have the EtherCAT based stack for controlling the BLDC motor. I plan to have 1 master & 1 slave and the BLDC motor is connected to the slave. At master, I would configure SOEM and at the slave, SOES. This way, the EtherCat stack/network would be created between the master and the slave which should be ok to control the BLDC motor.

In a nutshell.

Master(SOEM) -> Slave(SOES) -> BLDC Motor.

For explanation, we can think of Master as our laptop and slave as a Raspberry/Arduino/Any other board(might be attached to the driver separately) and BLDC motor would be connected with the slave.

I am looking for suggestions here to know if there is any technical deficiency in this architecture. I would be happy to share more details if needed.

Thank a lot in advance.

**Comment by ArthurKetels** [comment]
What you propose is a very common solution. There are already many products for sale that work exactly like this.

**Comment by bhargavah** [comment]
@ArthurKetels Thanks a lot for your reply.

I see. It would be excellent if you can comment on this architecture. 
Master(SOEM) -> Slave(SOES) -> Shield -> BLDC Motor

The shield is nothing but a board between Slave board & motor.

Also, it is good to know that such products are already there. It would be highly appreciable if you can point me out some links so that I can go ahead to try them.

**Comment by nakarlsson** [comment]
What exactly is missing? You need of the shelve hardware? Evaluation boards?

**Comment by bhargavah** [comment]
@nakarlsson Hi there, I did not get what you just said here.

**Comment by nakarlsson** [comment]
>It would be highly appreciable if you can point me out some links so that I can go ahead to try them.

Links to what? COTS HW that is ready to use? Or COST HW on which you can develop a SOEM -> SOES -> motor setup?

**Comment by dfreiberger** [comment]
Example COTS BLDC driver (combine with EK1100): https://www.beckhoff.com/en-us/products/i-o/ethercat-terminals/el7xxx-compact-drive-technology/el7211.html

Example DIY EtherCAT slave shield: https://www.bausano.net/en/hardware/ethercat-e-arduino/ethercat-and-raspberry.html

Or http://www.simplerobot.net/2018/06/rpi-ethercat-master-picat.html

**Comment by nakarlsson** [comment]
@bhargavah , examples given so we’ll close the issue
